packages/ruleset/src/rules/operation-summary.js

```diff
@@ -0,0 +1,15 @@
+const { oas3 } = require('@stoplight/spectral-formats');
+const { truthy } = require('@stoplight/spectral-functions');
+const { operations } = require('../collections');
+
+module.exports = {
+  description: 'Operation "summary" must be present and non-empty string.',
```

Comment:
```suggestion
  description: 'Operation "summary" must be present and a non-empty string.',
```

Reply:
I consciously worded this sentence this way because it follows the same pattern used by a couple of the spectral:oas rules.   Is that ok?